controllers/apply/awards-for-all/constants.js

```diff
@@ -17,7 +17,8 @@ const ORGANISATION_TYPES = {
     NOT_FOR_PROFIT_COMPANY: 'not-for-profit-company',
     SCHOOL: 'school',
     COLLEGE_OR_UNIVERSITY: 'college-or-university',
-    STATUTORY_BODY: 'statutory-body'
+    STATUTORY_BODY: 'statutory-body',
+    FAITH_GROUP: 'faith-group'
```

Comment:
This change (and the related ones) isn't strictly part of the a11y fixes but I thought it was easier to batch up with the others to avoid merge conflict hell later.